app/models/user.rb

```diff
@@ -18,6 +18,12 @@ class User < ActiveRecord::Base
   validates :email, uniqueness: true, presence: true, on: :create, email: true
   validates :email, email: true, on: :update
 
+  scope :for_review_counters, -> { joins(:cards).
+    select("id", "email", "count(cards.id) as for_review").
+    where("review_date < ?", Time.now).group("id") }
```

Comment:
Expression at 23, 52 should be on its own line.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.